src/assets/scripts/client/navigationLibrary/FixCollection.js

```diff
@@ -100,6 +100,24 @@ class FixCollection extends BaseCollection {
         return fixModel.relativePosition;
     }
 
+    /**
+     * Returns the pronunciation of the fix name if it exists, otherwise returns the input name
+     *
+     * @for FixCollection
+     * @method getFixSpokenName
+     * @param fixName {string}
+     * @return {string}
+     */
+    getFixSpokenName(fixName) {
+       const fixModel = this.findFixByName(fixName);
+
+       if (!fixModel) {
+           return fixName.toLowerCase(); // lower case to force speech engine to pronounce instead of spell
```

Comment:
Maybe this should be upper-case? How are strange fix names pronounced in lower-case? (comment for myself)

Reply:
I've experimented a bit, and it will spell fixes that are completely impossible to pronounce, for example `XFDCG`. However, other names, such as `ASDFG`, will have it try to pronounce them, which sounds super weird. I think I'll have to make it spell out all fixes that don't have a defined `spoken`.

@Fechulo: See this comment as the answer to our discussion on Slack. Too bad it didn't work out nicely! :sob:

Reply:
This is now changed.